File: 27/solution.py
from typing import List


class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        idx = 0
        count = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[idx] = nums[i]
                idx += 1
                count += 1
        return count

File: 27/solution.go
package _27

func removeElement(nums []int, val int) int {
	idx := 0
	count := 0
	for i := 0; i < len(nums); i++ {
		if nums[i] != val {
			nums[idx] = nums[i]
			idx++
			count++
		}
	}
	return count
}
